fgd/brush/trigger/zone_timer_stage.fgd

```diff
@@ -2,10 +2,11 @@
 = zone_timer_stage : "Starting zone for each stage of a map. zone_timer_start is automatically stage 1!" 
 	[
 	stage_name(string) : "Stage Name" : : "A custom name for this stage (optional)."
-	filter_name(string) : "Filter Name" : : "filter_activator entity to use for this zone."
+	filter_name(filterclass) : "Filter Name" : : "filter_activator entity to use for this zone."
 	stage_number(integer) : "Stage Number" : 2 : "The stage number of this zone. NOTE: 1 is reserved for start! 2+ are free for your stage zones, so start at 2!"
-	checkpoints_required(boolean) : "Checkpoints Required" : 1 : "Is the player required to activate all checkpoints (in order) before progressing to the next stage? Otherwise, checkpoint zones may be skipped for this stage."
-	limit_ground_speed(boolean) : "Limit ground speed" : 0 : "Limit ground speed to walking speed (prevents bunny hopping)."
+	checkpoints_required(boolean) : "Checkpoints Required" : 0 : "Is the player required to activate all checkpoints before progressing to the next stage? Otherwise, checkpoint zones may be skipped for this stage."
+	checkpoints_ordered(boolean) : "Checkpoints Ordered" : 1 : "Are the checkpoints placed in a particular order?"
+	limit_ground_speed(boolean) : "Limit ground speed" : 1 : "Limit ground speed to walking speed (prevents bunny hopping)."
```

Comment:
I defaulted this to true because I think it is more common to have stages that prevent bhopping.